### PR TITLE
Review contributing.md

### PR DESCRIPTION
**Pull request by Gokul2003g**
Few minor changes to improve readability and clarity.
Suggest more changes if any. 
Overall looks fine.
#470

### CODE REVIEW

**Comment by BaseMax** [comment]
@jbampton

**Comment by BaseMax** [comment]
Thanks, and welcome to the community.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <h1 align="center"><strong>Contributing to the ONE Programming Language</strong></h1>
 
 > Welcome new contributors! First of all thank you for contributing! :+1: :tada:
+<hr>
```
Comment (jbampton) [inline CONTRIBUTING.md:4]:
```suggestion
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,12 +10,11 @@
 The `ONE Programming Language` is a very welcoming community. We appreciate your
 ideas and your help. You need to communicate in
 [our discord server](https://discord.com/invite/sFCE2HcMCa) or pick an issue and
-share your perception of the solution you have. Don't forget to fork this
-repository and create a feature or new branch to work on your contribution.
+share your perception of the solution you have. Ensure you fork this repository. Create a new branch to work on your contribution for better code isolation.
```
Comment (jbampton) [inline CONTRIBUTING.md:13]:
```suggestion
share your perception of the solution you have. Ensure you fork this repository.
Create a new branch to work on your contribution for better code isolation.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -9,12 +10,11 @@
...
 
 ### **Then, make a PR**
 
-Now that you finished the work, don't forget again to run `pre-commit`. Create the
+Now that you finished the work, don't forget again to run `pre-commit`. `pre-commit` helps ensure code quality before creating a PR. Create the
```
Comment (jbampton) [inline CONTRIBUTING.md:17]:
```suggestion
Now that you finished the work, don't forget again to run `pre-commit`.
`pre-commit` helps ensure code quality before creating a PR. Create the
```

**Review by jbampton — APPROVED** [review]

**Comment by jbampton** [comment]
Thanks !!

**Pull request merged**